popup/popup.html

```
@@ -9,8 +9,32 @@
 
   <body>
     <div class="container">
-      <h2>Save to Notion</h2>
-      <p id="status">準備儲存</p>
+      <div class="popup-titlebar">
+        <h2 id="popup-title">Save to Notion</h2>
+        <div id="account-section" class="account-section" style="display: none">
+          <button id="account-button" class="account-button">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)